x/evm/types/msg_create_deploy_token.go

```diff
@@ -71,5 +72,14 @@ func (m CreateDeployTokenRequest) ValidateBasic() error {
 		return err
 	}
 
+	dailyMintLimit, err := sdk.ParseUint(m.DailyMintLimit)
+	if err != nil {
+		return err
+	}
+
+	if dailyMintLimit.IsZero() {
```

Comment:
why >0 here?

Reply:
It doesn't have to be >0 in the contract, but we've decided to make it the case in core when deploying just for safety reasons.